apps/pr-deploy-site/just.config.js

```diff
@@ -8,6 +8,9 @@ const findGitRoot = require('@uifabric/build/monorepo/findGitRoot');
 const gitRoot = findGitRoot();
 let instructions = [];
 
+// TODO: this doesn't appear to be working compared to master. why?
+// TODO: this works when 'yarn build' is called in 'apps/pr-deploy-site', but not when called from repo root.
+// TODO: is this meant to run purely as a side effect? when should it run? what triggers it? seems unreliable.
```

Comment:
@kenotron I'm looking into this now. I'm not sure why it's not working from root. Just wanted to tag you in case you see anything obvious. Thanks!

Reply:
I think I have to disable use of `.cached()` for all commands other than ` build`. The inputs to the other commands (`lint`, `test`, `bundle`) include library and dist output, and `.cached()` won't be reliable for detecting changes to those files.